crates/swc_css_minifier/tests/fixture/compress-calc/convert-units/output.min.css

```diff
@@ -1 +1 @@
-.class1{width:calc(1cm + 1px)}.class2{width:calc(1px + 1cm)}.class3{width:20Q}.class4{width:calc(100.9q + 10px)}.class5{width:calc(10px + 100.9q)}.class6{width:calc(10cm + 1px)}.class7{width:calc(1cm + 1px)}.class8{width:calc(10px + 1q)}.class9{width:401q}.class10{width:41q}.class11{width:calc(10in + 1q)}.class12{width:calc(10pt + 1q)}.class13{width:calc(10pc + 1q)}.class14{width:calc(1q + 10px)}.class15{width:401q}.class16{width:41q}.class17{width:calc(1q + 10in)}.class18{width:calc(1q + 10pt)}.class19{width:calc(1q + 10pc)}.class20{width:3unknown}.class21{width:calc(1unknown + 2px)}.class22{width:calc(1px + 2unknown)}
+.class1{width:1.0264583333333333cm}.class2{width:38.79527559055118px}.class3{width:20Q}.class4{width:111.48333333333333q}.class5{width:105.33858267716536px}.class6{width:10.026458333333334cm}.class7{width:1.0264583333333333cm}.class8{width:10.94488188976378px}.class9{width:10.025cm}.class10{width:1.025cm}.class11{width:10.009842519685039in}.class12{width:10.708661417322835pt}.class13{width:10.059055118110237pc}.class14{width:11.583333333333334q}.class15{width:401q}.class16{width:41q}.class17{width:1017q}.class18{width:15.11111111111111q}.class19{width:170.33333333333334q}.class20{width:3unknown}.class21{width:calc(1unknown + 2px)}.class22{width:calc(1px + 2unknown)}
```

Comment:
Sounds like we need option for this, can we implement this?

Reply:
Can you precise what do you have in mind? Something like a "enableMixedUnit" option? "allowSumOfMixedUnit" ?